pkg/streamingpromql/functions_test.go

```diff
@@ -80,6 +80,7 @@ func TestFunctionDeduplicateAndMerge(t *testing.T) {
 	expressions := map[string]string{
 		// Please keep this list sorted alphabetically.
 		"abs":                `abs({__name__=~"float.*"})`,
+		"absent":             `absent({__name__=~"float.*"})`,
```

Comment:
It's probably better to skip this for `absent` - `absent` doesn't need deduplication and merging:

```suggestion
		"absent":             `<skip>`,
```